### DISCUSSION TITLE
How do I find an address?

### DISCUSSION THREAD

**Discussion by roybrotman** [community/OSPO#33] [Tools]
Hi all, I am new so please forgive me.
All I want to know is how to find address number for a street in OSMandMaps .

most navigation programs have you enter an address as: 123456 Any Street, city(if needed)
That doesn’t work with OSMAndMaps
I try to enter an address an it plops me somewhere on the street or avenue and doesn’t give me the chance to enter a address number OSMandMaps nomenclature
being provided I have entered STREET name & number but most of the time it dumps me randomly around the streets but not to the correct address? What am I doing wrong?

**Comment by ahpook** [comment]
Hi @roybrotman - i think you've filed this in the wrong repo. Try https://github.com/osmandapp/OsmAnd/issues ! They don't have discussions enabled but you can file an issue.

**Reply by Matary1** [reply]
thank you for best support

**Comment by Matary1** [comment]
yes please say
